src/app/feature-flag/banner/feature-banner.component.less

```diff
@@ -0,0 +1,111 @@
+@import (reference) '../../../assets/stylesheets/shared/osio.less';
+
+.featureLevel(@color-for-feature) {
```

Comment:
@corinnekrych ideally, classes only use hyphens and IDs use camelCase text. Would you be able to update, or does this affect the overall feature?